### ISSUE TITLE
KG2E crashes after around 400 epochs

### ISSUE THREAD

**Issue by ArkDu** [Sujit-O/pykg2vec#183]
## Issue Template
First of all, thank you for finding the bugs. Please help us improve by providing the bug report as much as possible in the following format.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - SDK [e.g. pycharm]

**Dependency (please complete the following information):**
 - Current version of the dependencies

**Additional context**

**Comment by baxtree** [comment]
Hi, @ArkDu, I noticed the default learning rate for KG2E is 0.01, which may have been too high and resulted in a large gradient during the training. Would you like to try decreasing it and see if that helps?

**Comment by ArkDu** [comment]
Sure I'll try a smaller learning rate and see how it goes.
